form.php

```diff
@@ -31,6 +31,6 @@
 </div>
 
 </body><!--Connects javascript files with site-->
-  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
```

Comment:
I downloaded the file required for Jquery. This meant I had to link my site to the file rather than to the internet.